docs/digger/installation/prerequisites.asciidoc

```diff
@@ -0,0 +1,20 @@
+== Prerequisites
+
+A Digger server can be installed using the https://github.com/aerogear/digger-installer[Digger Installer tool].
+
+Before running the Digger installer, please make sure the following prerequisites are met:
+
+=== Ansible
+
+Ansible is installed locally. The minimum required version is 2.2. You can check http://docs.ansible.com/ansible/intro_installation.html[Ansible installation instructions].
```

Comment:
Maybe a small comment on how to check the current installed Ansible version/if ansible is installed already. `ansible --version`.